pkg/nfs/statefulset.go

```diff
@@ -147,4 +148,18 @@ func (d *Deployment) createPodTemplateSpec(nfsPort int, httpPort int, labels map
 			},
 		},
 	}
+
+	// If a PVC is provided in the NFSServer CR, add a reference to the PVC in
+	// the volumes list.
+	if d.nfsServer.Spec.PersistentVolumeClaim.ClaimName != "" {
+		pvc := corev1.Volume{
+			Name: PVCNamePrefix,
+			VolumeSource: corev1.VolumeSource{
+				PersistentVolumeClaim: &d.nfsServer.Spec.PersistentVolumeClaim,
+			},
+		}
+		podSpec.Spec.Volumes = append(podSpec.Spec.Volumes, pvc)
+	}
```

Comment:
I'm not sure about this, but would it make more sense to move this block as the "else" to the conditional at statefulset.goL36?  At the moment both sides of the conditional are in different places.

Reply:
Yes, that would be much better.